nextflow.config

```diff
@@ -0,0 +1,74 @@
+process {
+    tag = {"$sid"}
+    publishDir = {"./results_conn/$sid/$task.process"}
+    scratch = true
+    stageInMode = 'copy'
+    stageOutMode = 'rsync'
+}
+
+params {
+    help=false
+    root=false
+    template=false
+    labels_list=false
+    labels_img_prefix=''
+    output_dir="./results_conn/"
+
+    apply_t1_labels_transfo=true
+
+    // **COMMIT options**//
+        run_commit=true
+        b_thr=50
+        nbr_dir=500
+        ball_stick=false
+        para_diff="1.7E-3"
+        perp_diff="0.85E-3 0.51E-3"
+        iso_diff="1.7E-3 3.0E-3"
+
+    // **Decompose options**//
+        no_pruning=false
+        no_remove_loops=false
+        no_remove_outliers=false
+        min_length=20
+        max_length=200
+        loop_max_angle=330
+        outlier_threshold=0.5
+
+    // **AFD & RD options**//
+        run_afd_rd=true
+        length_weighting=false
+        sh_basis="descoteaux07"
+
+    use_similarity_metric=true
+    nbr_of_samples=100
```

Comment:
I would scale this down to 30 or so. 
Otherwise, the average_connection process is way too long...

Reply:
I reduce it to 50, I did some test with 10-25-50 and 50 seemed to be enough. 25 was ok for Freesurfer (big connection) but was not so great for Glasser (very small connection)